danger/Dangerfile.swift

```diff
@@ -0,0 +1,9 @@
+import Danger
+import Foundation
+
+let danger = Danger()
+
+// SwiftLint.lint(.modifiedAndCreatedFiles(directory: "Sources"), inline: true)
+SwiftLint.lint(inline: true, configFile: ".swiftlint.yml", strict: true)
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/62d27ff0e52e3e50c99aacadf8bdd80156ff444a/danger/Dangerfile.swift#L9"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>Dangerfile.swift:9</code></del></td>
    </tr>
  </tbody>
</table>